lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
```

Comment:
Maybe the best way is to declare `timer` variable without actually setting a value to it?